checker/tests/nullness-extra/shorthand/Makefile

```diff
@@ -1,23 +1,22 @@
-.PHONY: all  nullnessOnly nullnessRegex nullnessBad nonsense
+.PHONY: all
```

Comment:
Why remove all these targets? You should add `clean`, but the other targets should remain phony.